packages/react-native-editor/CHANGELOG.md

```diff
@@ -9,7 +7,7 @@ For each user feature we should also add a importance categorization label  to i
 [*] → Minor enhancements and fixes that address annoyances — things our users can miss.
 -->
 
-## Unreleased
+## To be released
```

Comment:
We can put here anything that isn't `Unreleased` so it is ignored by this code:

https://github.com/WordPress/gutenberg/blob/906407bb8fb7af332541f2462092b38d6895d3bc/bin/plugin/commands/common.js#L96-L104